Source/WebKitLegacy/win/WebFrame.cpp

```diff
@@ -1079,7 +1079,7 @@ Ref<Frame> WebFrame::createSubframeWithOwnerElement(IWebView* webView, Page* pag
     d->webView->viewWindow(&viewWindow);
 
     this->AddRef(); // We release this ref in frameLoaderDestroyed()
-    auto frame = Frame::create(page, ownerElement, makeUniqueRef<WebFrameLoaderClient>(this));
+    auto frame = Frame::create(page, ownerElement, makeUniqueRef<WebFrameLoaderClient>(this), WebCore::FrameIdentifier::generate());
```

Comment:
Maybe we could make the frameID parameter optional (and std::optional<>) and let the Frame constructor generate the frameID if necessary. I feel it would be nicer than having this explicit WebCore::FrameIdentifier::generate() as parameter.